recipes/klib/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "klib" %}
+{% set version = "0.0.69" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 17611d0416af42d034d3532c624fde45677af45a0adc8edb83dd1556979961df
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - matplotlib >=2.1.2
+    - numpy >=1.13.3
+    - pandas >=1.0.0
+    - seaborn >=0.1.0
+  run:
+    - matplotlib >=2.1.2
+    - numpy >=1.13.3
+    - pandas >=1.0.0
+    - python
+    - seaborn >=0.1.0
+
+test:
+  imports:
+    - klib
+    - klib.tests
+
+about:
+  home: "https://github.com/akanz1/klib"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "Customized data preprocessing functions for frequent tasks."
+  doc_url: 
+  dev_url: 
```

Comment:
`dev_url: "https://github.com/akanz1/klib"`

and remove the `doc_url` entry if there is none.